poster/styles/styles.scss

```diff
@@ -1,63 +1,477 @@
+    background-size: 50%;
+    grid-column: 3 / 4;
+    grid-row: 2 / 3;
+}
+
+.dot-background {
+    background: #212121 url("../images/dot_pattern.png");
+    background-size: cover;
     grid-column: 3 / 4;
     grid-row: 3 / 5;
-    background: #E2E2E2;
-    color: #212121;
 }
 
-.primary-details {
+.layer-pattern {
+    background: #212121 url("../images/layer-pattern.png");
+    background-size: cover;
+    margin-bottom: 1rem;
+}
+
+.dark-noise_pattern {
+    background: #212121 url("../images/dark_noise_pattern.png") repeat;
+    background-size: 50%;
+}
+
+// grids-flex //
+
+.primary-details_flex {
+    display: flex;
+    flex-direction: column;
     grid-column: 4 / 5;
     grid-row: 1 / 2;
-    background: #E0E0E0;
 }
 
-.secondary-details {
+.tertiary-details_flex {
+    display: flex;
+    flex-direction: column;
     grid-column: 4 / 5;
-    grid-row: 2 / 3;
-    background: #7E7E7E;
+    grid-row: 4 / 5;
 }
 
-.tertiary-details {
-    grid-column: 4 / 5;
+.names_flex {
+    display: flex;
+    flex-direction: column;
+    grid-column: 1 / 3;
     grid-row: 4 / 5;
-    background: #9E9E9E;
+    margin-top: 2rem;
 }
 
-.remove-list {
-    list-style: none;
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
+.flex-grid {
+    flex: 1;
+}
+
+// shapes // 
+
+.shape {
+    position: absolute;
+    z-index: 9000;
+
+    &--blue {
+        bottom: 120px;
+        left: 920px;
+        height: auto;
+        width: 15%;
+    }
+
+    &--red {
+        bottom: 320px;
+        right: 950px;
+        height: auto;
+        width: 15%;
+    }
+
+    &--purple {
+        top: 850px;
+        left: 450px;
+        height: auto;
+        width: 12%;
+    }
+
+    &--green {
+        bottom: 760px;
+        left: 650px;
+        height: auto;
+        width: 12%;
+    }
+
+    &--yellow {
+        bottom: 450px;
+        left: 450px;
+        height: auto;
+        width: 20%;
+    }
+}
+
+// shapes media query //
+
+@media screen and (max-width:1075px) {
+
+    .shape {
+        position: absolute;
+        z-index: 9000;
+
+        &--blue {
+            bottom: 420px;
+            left: 450px;
+            height: auto;
+            width: 15%;
+        }
+
+        &--red {
+            bottom: 150px;
+            right: 40px;
+            height: auto;
+            width: 15%;
+        }
+
+        &--purple {
+            top: 100px;
+            left: 520px;
+            height: auto;
+            width: 12%;
+        }
+
+        &--green {
+            bottom: 50px;
+            left: 350px;
+            height: auto;
+            width: 12%;
+        }
+
+        &--yellow {
+            bottom: 300px;
+            left: 120px;
+            height: auto;
+            width: 20%;
+        }
+    }
+}
+
+// media query 1 //
+
+@media screen and (max-width: 766px) {
+
+    body {
+        grid-template-columns: repeat(3, 33.3%);
+        grid-template-rows: repeat(5, 255px);
+        position: relative;
```

Comment:
See notes above about media queries. You don't need to repeat rules you've already set, like `position: relative;`